### ISSUE TITLE
Documentation help wanted

### ISSUE THREAD

**Issue by Vimux** [Vimux/Mainroad#281]
I'm not that good at explaining and writing, so I need help with documentation.

The overall idea is to fix typos and errors, simplify and streamline the content, add missing parts(?). Key point: don't
describe what Hugo can do, describe what Mainroad theme can do. In other words, our documentation will mainly cover
things unique to Mainroad.

* [exampleSite/content/docs/getting-started.md](https://github.com/Vimux/Mainroad/blob/master/exampleSite/content/docs/getting-started.md)
* [exampleSite/content/docs/customization.md](https://github.com/Vimux/Mainroad/blob/master/exampleSite/content/docs/customization.md)
* * [exampleSite/content/docs/faq.md](https://github.com/Vimux/Mainroad/blob/master/exampleSite/content/docs/faq.md)
* [README.md](https://github.com/Vimux/Mainroad/blob/master/README.md)
* [CONTRIBUTING.md](https://github.com/Vimux/Mainroad/blob/master/CONTRIBUTING.md)

If someone has (technical) writing skills and wants to help me out there, any help is much appreciated!

P.S. As always, I'm open to any feedback on the overall plan or details.

**Comment by Ashwani-Verma-07** [comment]
hi ,@Vimux  can you assign this issue to me

**Comment by Vimux** [comment]
@Ashwani-Verma-07,

No problem, I can assign you to this issue, but please let me know what you plan to do: fix typos, rewrite something, add missing parts? What files do you want to work on which are listed above in the issue description?

Other contributors and I will be able to take this into account and plan our future tasks accordingly.

**Comment by Ashwani-Verma-07** [comment]
yes i will look out all errors related to fix typos, grammatical error on **exampleSite/content/docs/faq.md**

**Comment by lisa005** [comment]
Hello! I'm fairly new here, but I'd like to help edit the Getting Started and FAQ sections for conciseness, grammar, and typos.

**Comment by Vimux** [comment]
Good, any help is welcome. I assign you.

Tip: read [CONTRIBUTING#pull-requests-pr](https://github.com/Vimux/Mainroad/blob/master/CONTRIBUTING.md#pull-requests-pr).

**Comment by cryptotechblog** [comment]
Can someone add what is the proper way to create multiple empty lines in a .md? I.e. I'm typing here 



and now I'm typing here? Unless I missed something up with the theme nothing I do in my .md files will allow it.

**Comment by InvestorPrivacy** [comment]
Forgive me. I am not a developer by trade and have spent less than a week learning hugo and Mainroad. I have run into some minor challenges putting together a basic site with the Mainroad theme. I realize your faq mentions needing intermediate to advanced or developer knowledge of Hugo but with minimal work the documentation could help a beginner like me stumble much less. I have a few questions and suggestions. I am very new to using Github so please let me know if this is not a good place to post this.

1. 
Can someone add what is the proper way to create multiple empty lines in a post (.md) with mainroad? I.e. I'm typing here
&nbsp;
&nbsp;
&nbsp;
and now I'm typing here? Unless I missed something up with the theme nothing I do in my .md files will allow it i.e.   Most markdown formatting seems to work otherwise.

2. 
On the basic html elements page, what are you trying to explain to the reader? If I look at the basic html elements page in the examplesite directory I can clearly see the formatting needed to achieve the samples shown. If the intent is to show the reader how to format blog posts, I can suggest one of two things.  Easier / less work - Add a note at the start of the page to direct the reader to the raw file in the examplesite directory so they can view the proper syntax of how to format posts with Mainroad. More work - Explain all of the formatting requirements in plaintext before rendering them so the reader does not have to look into the source file.

3. I believe Mainroad ignores html elements in .md files (posts). If this is true, I believe this should be stated in either FAQ or basic elements, or both.

4. I could suggest renaming Basic Elements to match the post title "Basic HTML Elements" as the other posts do. I realize this is not the proper way to name posts for SEO but it makes slightly more sense to a reader with less skills than "Basic elements".

5. Getting started - The explanation of the config is a bit confusing and part of it is me reading too fast/skimming/looking in the source instead of the getting started, and maybe there is a good reason. I started with the [demo config](https://github.com/vimux/mainroad/blob/master/exampleSite/config.toml) that comes in the git and it took me a few weeks to realize that all of the configurations were included in [example config](https://github.com/Vimux/Mainroad#configtoml-example), also listed on that page and then again on the customization page. Documentation would strongly benefit from defining which configurations are required, and which are optional. 

6. Getting started - It would be more clear if the Getting started page was renamed "Getting started with Mainroad" as the first post ever is already a named "Getting started with Hugo". The lead text does clarify this, however.

7. [configtoml-example](https://github.com/vimux/mainroad#configtoml-example) internal templates (Open Graph, Schema, Twitter Cards, Disqus, Google Analytics) - Probably obvious to developers but took me time to figure out what Open Graph and Schema were. If these are default with Hugo this is mainly my lack of Hugo knowledge and development experience. 
7b. A lot of the configtoml examples could stand to be elaborated on further - i.e. home = right, list = left, single = false. What can these be changed to and what does changing them do? If this is default Hugo functionality, please ignore. 


I really like the theme and hugo is also really neat. It's absolutely amazing that you can host a basic site for only the cost of a domain. I have much to learn about how to ensure content is displayed exactly the way I want it to through CSS or whatever  as text only is too barebones for my content. Thanks a lot for putting in all of the hard work and effort. I've written a lot of technical documentation in my time as a non-developer, please let me know if I can help in any way to pay it back / forward.

**Comment by rmcf** [comment]
Hi @Vimux I am grateful for Mainroad theme and would like to suggest Ukrainian localization for user interface. Is it interesting for you? Also, I have a question about “search” widget – should I create an issue or may I ask it here? I am very new in Hugo and don’t know how to configure it correctly to search my posts by title.

**Comment by Vimux** [comment]
@rmcf,

> I am grateful for Mainroad theme and would like to suggest Ukrainian localization for user interface.

Go ahead. Please read CONTRIBUTING.md first. It will help you avoid most problems.

I may be slow to respond in the coming days, but I will look into it as soon as I have time.

> Also, I have a question about “search” widget – should I create an issue or may I ask it here? I am very new in Hugo and don’t know how to configure it correctly to search my posts by title.

The widget redirects the user's query to the search engine, limiting the search to the site. I do not provide free technical support, so your specific personal question will remain unanswered no matter which of these two actions you take. Our issues are for bug reports and feature requests only.

**Comment by richard-salam** [comment]
Hey @Vimux  I'd love to help with any technical documentation. :)

**Comment by Vimux** [comment]
@richard-salam,

Better documentation is a good thing. Do you have something in mind? It would be better if we discussed it briefly before you invest your time.

**Comment by richard-salam** [comment]
> @richard-salam,
> 
> Better documentation is a good thing. Do you have something in mind? It would be better if we discussed it briefly before you invest your time.

I agree @Vimux  I am available on LinkedIn https://in.linkedin.com/in/richardsalame  Please ping me there, let us discuss :)

**Comment by BboyGT** [comment]
Hi, I am interested in helping with documentation improvements if this is still open to contributors.

I am a software engineer and technical writer. I can help simplify and streamline docs, clarify what is unique to Mainroad, improve README/CONTRIBUTING flow, and make the getting started/customization/FAQ pages easier for new users to follow.

GitHub: https://github.com/BboyGT
X: https://x.com/GodstimeDev
Email: masathygolden@gmail.com

If useful, I can start with one focused PR for a single docs page.
